person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
```

Comment:
- [ ] Consider making sure the id should only be able to be retrieved so only the Getter has defined not the setter.